artnet_receiver/artnet_receiver.ino

```diff
@@ -0,0 +1,44 @@
+#define ARTNET_ENABLE_WIFI true
+
+#include "../lib/homie.h";
+#include "artnet/Artnet.h";
+using namespace std;
+
+//homie.
+Homie homie; 
+//Arturo, Artnet Specialist
+ArtnetWiFiReceiver arturo; 
+
+int universe = 1;
+
+void pixelmap_receiver(uint8_t* data, uint16_t size)
+{
+  if (size % 3 != 0) {
```

Comment:
Assuming 3 channel pixel mapped data. 5 channel exists and this should be configurable when you instantiate Arturo